app/javascript/react/components/Artists/ArtistsContainer.js

```diff
@@ -22,13 +28,34 @@ const ArtistsContainer = (props) => {
     getArtists();
   }, [artists.length]);
 
+  const deleteArtist = async (id) => {
+    try {
+    const response = await fetch(`api/v1/artists/${id}`, {
+      method: "DELETE",
+      credentials: "same-origin",
+      headers: {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+      },
+      body: JSON.stringify(),
+      })
+      if (response.status === 204) {
+        getArtists()
+      }
+      } catch (error) {
+        console.log(error)
+      }
```

Comment:
- something about this identation is off, I think the if statement!

- we shouldnt be calling getArtists again. Instead, you'll need to use a `filter()` method to delete the review from state (after the fetch request). You still have the id of the old review, so use that to find the review in state and get it out!